docs/diagrams/ListPoolSequenceDiagram.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant "lp:ListPoolCommand" as ListPoolCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute(command)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(command)
+activate AddressBookParser
+
+AddressBookParser -> ListPoolCommand
```

Comment:
I think there should be a create ListPoolCommand here as the AddressBookParser returns a new ListPoolCommand object?